pkg/driftctl_test.go

```diff
@@ -49,24 +48,13 @@ func runTest(t *testing.T, cases TestCases) {
 			remoteSupplier := &resource.MockSupplier{}
 			remoteSupplier.On("Resources").Return(c.remoteResources, nil)
 
-			var filter *jmespath.JMESPath
-			if c.filter != "" {
-				f, err := filter2.BuildExpression(c.filter)
-				if err != nil {
-					t.Fatalf("Unable to build filter expression: %s\n%s", c.filter, err)
-				}
-				filter = f
-			}
```

Comment:
Given the verbosity to rewrite in each test case the filtering option, I wonder if it wouldn't be better to just add the boolean for the strict mode instead.

Reply:
Yes it would be less verbose but it adds complexity when it comes to use other ScanOption fields, isn't it? I think this approach gives more control on what we want to test without modifying the test case struct.